ocrvs 5546

sadmananik opened the pull request.

Zangetsu101 reviewed (commented):
Nice work @sadmananik. We will need to take another look at fhirIdMapping and the new fhirIdPatient.
Another would be adding a data source for fetching the patient resource.

sadmananik commented:
> Nice work @sadmananik. We will need to take another look at fhirIdMapping and the new fhirIdPatient. Another would be adding a data source for fetching the patient resource.

i have added datasource and also fixed _fhirIDPatient.
